test/ctia/http/routes/judgement_test.clj

```diff
@@ -80,13 +82,45 @@
       (testing "GET /ctia/judgement/search"
         ;; only when ES store
         (when (= "es" (get-in @ctia.properties/properties [:ctia :store :indicator]))
-          (let [term "observable.value:1\\.2\\.3\\.4"
+          (let [term "observable.value:\"1.2.3.4\""
                 response (get (str "ctia/judgement/search")
                               :headers {"api_key" "45c1f5e3f05d0"}
                               :query-params {"query" term})]
             (is (= 200 (:status response)))
             (is (= "Malicious" (first (map :disposition_name (:parsed-body response))))
-                "quoted term works"))
+                "IP quoted term works"))
+
+          (let [term "1.2.3.4"
+                response (get (str "ctia/judgement/search")
```

Comment:
Take a look at the destructuring on line 31.  It makes these tests a little more concise.